client/src/components/SearchBox/SearchBox.styles.scss

```diff
@@ -44,8 +44,10 @@
         width: 100%;
   
         z-index: 50;
-        // copied box-shadow from chakra UI shadow 2xl
-        box-shadow: 0 25px 50px -12px rgba(0, 0, 0, 0.25);
+        margin-inline-start: 0;
+        // copied box-shadow from chakra UI shadow 2xl,
+        // with color from dark-lg
+        box-shadow: 0 25px 50px -12px rgba(0, 0, 0, 0.6);
```

Comment:
is the color from the dark-lg [here](https://github.com/chakra-ui/chakra-ui/blob/75edcf41e7ff4acc2569f2169949063c164d8f6e/packages/theme/src/foundations/shadows.ts)? those dark-lg values look slightly different to me

Reply:
Yea okay, they're not, I'll adjust the comment accordingly